<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1754</width>
    <height>959</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>301</width>
      <height>851</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Logic Structure</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>Linear_Structure</string>
     </property>
     <item>
      <property name="text">
       <string>Linear_List</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Stack</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Skip List</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>Tree_Structure</string>
     </property>
     <item>
      <property name="text">
       <string>Binary_Tree</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tree</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>Graph_Structure</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>1411</width>
      <height>841</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>90</y>
           <width>601</width>
           <height>201</height>
          </rect>
         </property>
         <property name="text">
          <string>Welcome</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="0,2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>{Selected Structure}</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_selected">
             <property name="text">
              <string>{Structure Class Name}</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_clear_console">
             <property name="text">
              <string>Clear Console</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextBrowser" name="console">
           <property name="focusPolicy">
            <enum>Qt::FocusPolicy::NoFocus</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QPushButton" name="button_back">
             <property name="text">
              <string>Back</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="button_redo">
             <property name="text">
              <string>Redo</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QTextBrowser" name="textBrowser_2"/>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QPushButton" name="button_export">
             <property name="text">
              <string>Export</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1754</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
